documentation/src/main/asciidoc/topics/contributing/basics.adoc

```diff
@@ -3,14 +3,14 @@ In this chapter we quickly walk through the basics on contributing; future chapt
 
 == Pre-requisites
 |=============
-| Java 11   | {brandname} is baselined on Java 8 but needs to be built with Java 11 (we build with OpenJDK)
+| Java 11   | {brandname} is baselined on Java 11 but needs to be built with Java 11 (we build with OpenJDK)
```

Comment:
```suggestion
| Java 11   | {brandname} is built using and requires Java 11 for runtime (we build with OpenJDK)
```
Or something similar. The wording with baselined and built with above sounds a bit weird with the same version.